src/Microsoft.Azure.WebJobs.Host/Loggers/ValueWatcher.cs

```diff
@@ -15,120 +15,4 @@
 
 namespace Microsoft.Azure.WebJobs.Host.Executors
 {
-    internal class ValueWatcher
```

Comment:
How can you remove this? This is what handles periodic updates of live parameter logs in the Dashboard for long running functions, based on a background timer. It looks now like you only update parameters once on Flush.

I'm nervous about messing around with this code too much, since for Dashboard stuff our main line of defense is our Dashboard UI tests. If you're changing this stuff you should run them.